jupyter_scheduler/pydantic_v1/main.py

```diff
@@ -0,0 +1,4 @@
+try:
+    from pydantic.v1.main import *
+except ImportError:
+    from pydantic.main import *
```

Comment:
(not specific to this file, just need a target per review comment)

We should add `dataclasses.py` in case it is needed in the future. See LangChain's `pydantic_v1` module source for reference: https://github.com/langchain-ai/langchain/tree/master/libs/langchain/langchain/pydantic_v1